### ISSUE TITLE
Use as Assist app

### ISSUE THREAD

**Issue by shuvashish76** [cvzi/ScreenshotTile#118]
Long press the home button to share your screenshots.
 - Permission : `Assist app`
Enable previews
 - Permission : `Display over other apps`

Refer : [Screenshot sharer](https://github.com/rosenpin/Screenshot-Sharer) app

**Comment by cvzi** [comment]
Sorry I don't understand what you want to say

**Comment by shuvashish76** [comment]
Sorry for the late reply. 

A clear and concise description of what the problem is. Ex. I'm always frustrated when

- I open notification drawer cause my device is a 8inch tablet. Its not very convenient to drag the notification bar in bigger size devices or tablets.
_(Plus I like dark theme but as my device is android 8.1.0 the notification bar is white by default & not possible to make it dark.)_
- It's possible to take screenshot using "Widgets", your (ScreenshotTile) app offers but problem is the user need to comeback to the home screen 1st to click widgets.

The app I mentioned in my 1st comment takes screenshot by simply long clicking the "Home" button from device navigation bar. You don't need to open notification drawer or go to launcher home screen, you can take screenshots while on any app.
It works as an assist app to do that.
I'd suggest you to take a look at [Screenshot sharer](https://github.com/rosenpin/Screenshot-Sharer) app & see how it works.

_(btw I love your app as it's able to take screenshots of notification drawer also)_

Thankyou🙏🏿

**Comment by cvzi** [comment]
Thanks I understand it now.
One question though: why not use that "screenshot sharer" app? Does it have a problem? 
-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

**Comment by shuvashish76** [comment]
> why not use that "screenshot sharer" app? Does it have a problem?

Cause none of those other apps or my system screenshot tool allow me to take screenshots of device notification drawer. Which is sometimes useful _(for example: to show notification related music/video player issue to devs)_ & I don't prefer keeping multiple similar category apps _(in this case screenshot apps)_

**Comment by cvzi** [comment]
It is now available in the new version [1.16.0](https://github.com/cvzi/ScreenshotTile/releases/tag/v1.16.0)

**Comment by shuvashish76** [comment]
Thanks normal screenshots by long pressing the home button works 👍🏿

Drag down your notification bar > long press home button. Why it miss those parts?

> Cause none of those other apps or my system screenshot tool allow me to take screenshots of device notification drawer.

But when I use "Delay" mode it captures notification bar...

I'm on android 8.1.0

**Comment by cvzi** [comment]
Does that mean it closes the notifications when you long press the home button?

**Comment by shuvashish76** [comment]
> Does that mean it closes the notifications when you long press the home button?

No. It just ignores the notification drawer & captures everything behind it.

**Comment by cvzi** [comment]
I didn't use the same method to take the screenshot as  with the quick settings or the floating button 😐

Probably it only works with the "original method" (the one that shows the cast icon) on your phone. I did not add that method to the home button, but I can probably add it. 

There is a new entry in the app settings "Assistant action" where you can choose what happens when you press the home button. You can try the other to methods there, but I doubt it will work.
-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.
